src/components/flowchart/index.js

```diff
@@ -216,6 +216,11 @@ export class FlowChart extends Component {
           [width + margin + metaSidebarWidth / scale, height + margin]
         ]);
 
+        // Ensure valid x and y values before performing zoom operations
+        if (!isFinite(x) || !isFinite(y) || isNaN(x) || isNaN(y)) {
+          return;
+        }
+
```

Comment:
Hey do you mind moving this up to the top of the function (immediately after line 202)?